lib/cfg-lexer.h

```diff
@@ -85,7 +85,12 @@ typedef struct _CfgLexerKeyword
 
 #define CFG_KEYWORD_STOP "@!#?"
 
-
+/* structure that describes a given file path type */
+typedef struct _CfgFilePath
+{
+  const gchar *path_type;
+  const gchar *file_path;
```

Comment:
Because of `file_path` is stored by `g_strdup`, this shouldn't be `const gchar *`, just `gchar *`.
As `path_type` is also dynamically allocated, it should also be `gchar *`.